MVVM Module/Source/Feed Presentation/FeedViewModel.swift

```diff
@@ -20,14 +20,19 @@ final class FeedViewModel {
 
 	var onLoadingStateChange: Observer<Bool>?
 	var onFeedLoad: Observer<[FeedImage]>?
+	var onFeedLoadError: Observer<Error>?
```

Comment:
`onFeedLoadError: Observer<Error>?` could be `onErrorStateChange: Observer<String?>?`, so the ViewModel decides when to show/hide errors (instead of the view controller). It's easier to manage presentation state transitions when all changes are centralized.

For example, in the `loadFeed()` method, the ViewModel can call  `onErrorStateChange(.none)` to hide the error before loading the feed.

This way, the ViewController doesn't need to decide when to hide the error, nor which message to display.
```suggestion
	var onErrorStateChange: Observer<String?>?
```

Reply:
it is implemented